jaxrs-api/src/main/java/jakarta/ws/rs/core/EntityPart.java

```diff
@@ -211,7 +213,7 @@ static Builder withFileName(String partAndFileName) {
      * 
      * @since 3.1
      */
-    public interface Builder {
```

Comment:
Why removing `public`?

Reply:
It's redundant for inner interfaces, as it is for methods etc. in all interfaces.